tests/integration/test_users.py

```diff
@@ -174,7 +174,7 @@ def test_patch_user(client, auth_headers, new_user, create_admin_user):
 
     newEmail = "unauthorizedpatch@test.com"
 
-    unauthorizedResponse = client.patch(f"/api/user/{userToPatch.id}", json={"email": newEmail}, headers=auth_headers["pm"])
+    unauthorizedResponse = client.patch(f"/api/user/{userToPatch.id}", json={"email": newEmail}, headers=staff_header)
```

Comment:
Good catch. It shouldn't matter since it is ultimately testing wether a non-admin is not allowed, but to be consistent with the previous test it should be. I changed this because the jwt's being used are unrealistic of actual jwts and causing an issue. Let's fix this.